### PR TITLE
Add post about team orange in mallorca

### PR DESCRIPTION
**Pull request by dataduke**

### CODE REVIEW

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "A team of ePagees conquers Tierra Buena in Mallorca"
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: The title is very nice but I think it just fits for internal readers. Maybe you find a SEO-friendly one that also attracts external readers?

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+date: "2016-03-16 06:16:03"
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: Change date to 2016-03-17.

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+icon: suitcase
+categories: working-at-epages
+authors: ["Alejandra A., Benjamin N. and Heiko A."]
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: Please just use your first names without the initial of your last names.

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "A team of ePagees conquers Tierra Buena in Mallorca"
+date: "2016-03-16 06:16:03"
+icon: suitcase
+categories: working-at-epages
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: Please also add the category "events".

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+authors: ["Alejandra A., Benjamin N. and Heiko A."]
+---
+
+Last week Team Orange, the ePages release and test automation team, was welcomed by the precious views, hearty people and refreshing atmosphere of Mallorca. The purpose of this particular trip was to celebrate an off-site team event liberated from the daily tasks and routines of the business life. The team set off to new shores to develop a master plan on how it could face the winds of change of the advancement of each team member. The ultimate goal was to find the best way for contributing to the promising future of the whole research and development department.
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: Last week Team Orange, the ePages release and test automation team,... > +Last week, the ePages release and test automation team, ...

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+Now we are back and ready to briefly tell you the tale of the fearless developers' journey to a mysterious island. So please enjoy the ambigous report!
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: So please enjoy the ambigous report! > Enjoy this ambiguous report.

What exactly would you like to express with "ambiguous"? Might sound a bit "ironic"?

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+{% image blog/blog-mallorca-1.jpg %}{% endimage %}
+
+With much curiosity we first took some time and visited the Drach Caves and the beautiful beach at Porto Cristo and recharged ourselves with inspiration.
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: With much curiosity we first took some time > With much curiosity we first took some time off

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+We also visited Sa Calobra and were delighted by the majestic mountains and the bluish tonalities of the Mediterranean Sea.
+
+{% image blog/blog-mallorca-2.jpg %}{% endimage %}
+
+Sharing daily life activities such as having breakfast, shopping, cooking, travelling by car and plane, playing board games and even doing sports together was definitely a delightful way to become closer. Our off-site time was a **BIG** chance to gain trust into the capabilities of our team. We totally adopted this magical present. Unsurprisingly, we soon recognized that living the team spirit can be a lot of fun!
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: off-site > offsite

Please change in the whole document.

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+{% image blog/blog-mallorca-3.jpg %}{% endimage %}
+
+There is always time for everything and so we had enough time to work on our technological and business challenges, too. We accomplished two knowledge sharing sessions, one by our pipeline expert in order to understand how it works, another one on language handling and language packs.
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: There is always time for everything and so we had enough time to work on our technological and business challenges, too.  > During our time on Mallorca, we of course had lots of time to work on our technological and business challenges, too.

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+{% image blog/blog-mallorca-4.jpg %}{% endimage %}
+
+We also played two rounds of [Coding Dojo](http://codingdojo.org), where we focused on solving the Kata of Conway’s Game of Life. In the first round we tried to apply the constraints “TDD as if you meant it” and “Driver & Navigator“, which strictly seperates the roles in pair programming. The second round focused on shaping distinct, comprehensive and sustainable test cases. Therefore, one ePagee had to implement an unit test. Next, the coding partner should lean into the position of an evil programmer and hence only implement as little as possible to make the test pass. Afterwards he gives back the keyboard to the radically stressed writer of the unit test, who has to enforce the desired implementation with the addition of another test case. In this rotary principle we trained our sense for writing high quality tests. This hands-on approach delivered a lot of amusements as well as practical benefits for our programming skills.
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: seperates > separates

...had to implement an unit test. > ... had to implement a unit test.

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+
+{% image blog/blog-mallorca-5.jpg %}{% endimage %}
+
+At one day a hackathon accelerated our ingenious chains of thoughts. From the early afternoon until late into the night three pairs took the opportunity to delve into topics of their interest. One pair evaluated the Jenkins [Pipeline Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Plugin) and built a prototype with multiple continuous delivery stages. Others automated the handling around our translation process. The third pair developed a new concept how to deal with failing tests – of course not without a working prototype. Presenting the results to the other pairs the next morning showed how quick ideas can grow into applicable solutions.
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: At one day a hackathon accelerated our ingenious chains of thoughts. > We also planned one day for a hackathon, that accelerated...


...three pairs took the opportunity to delve into topics of their interest. > three pairs of programmers took the opportunity to delve into topics of their interest.

File: _posts/2016/2016-03-16-team-orange-mallorca.md (hunk 6 above)
Comment (dataduke) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: vielfältig

File: _posts/2016/2016-03-16-team-orange-mallorca.md (hunk 10 above)
Comment (dataduke) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: ist das quasi ein stummes `u` - deswegen nur `a`, also wie im franz.?

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+
+Two [Design Thinking](http://hpi.de/school-of-design-thinking.html) sessions took us beyond the mental barrier of longsome beaten paths. The consistent execution of this method over and over again rapidly helped us to find the wanted ideas for our current team mission. Firstly, we want to return a lot of responsibilities to those teams that strongly depend on us in these days. Secondly, we want to establish a fast and secure mechanism to permanently keep the latest build of the next ePages version in a releasable state.
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: ...the wanted ideas... >... the desired ideas...

...depend on us in these days. > depend on us these days.

Secondly, we want to establish a > Secondly, we'd like to establish a...

latest build of the next ePages version > does that refer to ePages 6 software? or is  that an "outlook"?

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+
+{% image blog/blog-mallorca-6.jpg %}{% endimage %}
+
+In retrospective, Tierra Buena (“Good Land”) was indeed a very good place for thinking out of the box, reviving the technological perspective for each team member and filling the last gaps in our built-on automation landscape. We are more than happy to tell you that not only we have returned to Jena fueled by visionary understanding but we also conjured up a brand new continuous delivery pipeline mascot, Wall-E - self-assembled in Lego of course.
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: ....a very good place for thinking out of the box > to come up with new ideas and to think out of the box, ...

We are more than happy to tell you that not only we have returned to Jena fueled by visionary understanding but we also conjured up a brand new continuous delivery pipeline mascot, Wall-E - self-assembled in Lego of course. > We have returned to the office fueled by visionary understanding, but we also conjured up a brand new continuous delivery pipeline mascot, Wall-E - self-assembled in Lego, of course.

File: _posts/2016/2016-03-16-team-orange-mallorca.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+Overall, we can only recommend this welding together experience for every professional team. Such intense moments can sharpen your focus and broaden your sight at once, but most importantly keep you ahead of the competition.
```
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: Overall, we can only recommend this welding together experience for every professional team... > To sum up, we can definitely recommend this team-bonding experience for every professional team.

File: _posts/2016/2016-03-16-team-orange-mallorca.md (hunk 10 above)
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: yes :-)

File: _posts/2016/2016-03-16-team-orange-mallorca.md (hunk 6 above)
Comment (BirgitBader) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: Enjoy this ambiguous report. > Enjoy this multifaceted report.

File: _posts/2016/2016-03-16-team-orange-mallorca.md (hunk 4 above)
Comment (dataduke) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: add or switch?

File: _posts/2016/2016-03-16-team-orange-mallorca.md (hunk 12 above)
Comment (dataduke) [inline _posts/2016/2016-03-16-team-orange-mallorca.md]: exactly.. the epages 6 platform

File: _posts/2016/2016-03-17-team-orange-mallorca.md [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+
+{% image blog/blog-mallorca-5.jpg %}{% endimage %}
+
+## Inspiring one another
+
+We also planned one day for a **hackathon**, that accelerated accelerated our ingenious chains of thoughts. From the early afternoon until late into the night three pairs of programmers took the opportunity to delve into topics of their interest – which fortunately often matched the team's scope of duties. One team evaluated the [Jenkins Pipeline Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Plugin) and built a rapid prototype with multiple continuous delivery stages. Others engineered the ground-breaking steps for an **automated translation process**. The third team developed a new concept on how to deal with failing tests – of course not without a working prototype that considered a **quarantine test group**. Presenting the results to the others the next morning showed how quick ideas can grow into applicable solutions.
```
Comment (hahnert) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: Pls resolve the doubled "accelerated".

**Comment by dataduke** [comment]
old

**Pull request closed without merging**